assets/scss/layouts/footer/_footer.scss

```diff
@@ -144,17 +144,29 @@
 }
 
 .footer-payment-icons {
+    display: flex;
+    flex-wrap: wrap;
     height: remCalc(40);
```

Comment:
looks like we should remove fixed height as now list items can be out of list when we have more than one line